IDE/Espressif/ESP-IDF/examples/wolfssl_client/main/client-tls.c

```diff
@@ -42,25 +40,115 @@
     #include <wolfssl/wolfcrypt/mem_track.h>
 #endif
 
-const char *TAG = "tls_client";
+#ifdef WOLFSSL_ESPWROOM32SE
+static const char* TAG = "tls_client";
+#else
+const char*  TAG = "tls_client";
+#endif
+
+static void ShowCiphers(void)
+{
+    static char ciphers[4096];
+
+    int ret = wolfSSL_get_ciphers(ciphers, (int)sizeof(ciphers));
+
+    if (ret == WOLFSSL_SUCCESS)
+        printf("%s\n", ciphers);
+}
+
+#if defined(WOLFSSL_ESPWROOM32SE) && defined(HAVE_PK_CALLBACKS) \
+                                  && defined(WOLFSSL_ATECC508A)
+
+#include "wolfssl/wolfcrypt/port/atmel/atmel.h"
+
+int atcatls_set_callbacks(struct WOLFSSL_CTX* ctx);
```

Comment:
This function prototype is inwolfssl/wolfcrypt/port/atmel/atmel.h, so you should not need this here.

Reply:
Removed.